traitlets/config/configurable.py

```diff
@@ -365,9 +365,9 @@ def c(s):
                 # Truncate help to first line + "See also Original.trait"
                 if trait.help:
                     lines.append(c(trait.help.split('\n', 1)[0]))
-                lines.append('#  See also %s.%s' % (defining_class.__name__, name))
+                lines.append('#  See also: %s.%s' % (defining_class.__name__, name))
 
-            lines.append('# c.%s.%s = %s' % (cls.__name__, name, default_repr))
+            lines.append('#c.%s.%s = %s' % (cls.__name__, name, default_repr))
```

Comment:
Is it possible to keep it?

Otherwise, then the last code line does not differ from comments immediately above, and 
it needs 2 key strokes to enable the code.

Note, it was like these before #385 (actually i believe it was me that had passed this change in the first place, and had been accepted).